CONTRIBUTING.md

```diff
@@ -1,51 +1,60 @@
+* Contain comprehensive test coverage
 * Subject to vulnerability scan
-* Will be peer reviewed
 
-If source code does not meet all of the above criteria please place it in a different Orange Button repository.  It is fine to start code elsewhere and add it to the Core repository at a future date.
+# Getting started
 
-# Ground Rules
+You must sign the SunSpec Contributor License Agreement before enhancing the source code or the documentation.
+Please see the instructions on how to submit a signed CLA on the [Orange Button Open Source Community web page](https://sunspec.org/ob-open-source-community/).
 
-Responsibilities
+Also, please email support@sunspec.org or post a message on the [Orange Button Slack Channel](https://orange-button.slack.com/)
+so we are aware of your intent to contribute.
 
-* Follow the notes in the overview.
-* Create issues for any major changes and enhancements that you wish to make. Discuss things transparently and get community feedback.
-* Keep feature versions as small as possible, preferably one new feature per version.
-* Be welcoming to newcomers and encourage diverse new contributors from all backgrounds.  Being a library programmed in Python the [Python Community Code of Conduct](https://www.python.org/psf/codeofconduct/) is followed.
+For all changes follow the procedure documented at the bottom of the
+[Orange Button Open Source Community web page](https://sunspec.org/ob-open-source-community/).
 
-# Your First Contribution
+## Your First Contribution
 
-Unsure where to begin contributing to Orange Button Core?  Please enquire on our [Slack Channel](https://orange-button.slack.com/).
+Unsure where to begin?  Please enquire on our [Slack Channel](https://orange-button.slack.com/).
 
-# Getting started
+## How to report a bug
 
-You must sign the Orange Button Individual or Corporate Contributors agreement before enhancing or changing the documentation.  Also please send an email to support@sunspec.org or post a message on the Orange Button developers slack channel so we are aware of your intent to contribute.
+Please use [GitHub Issues](https://github.com/SunSpecOrangeButton/pyoblib/issues) for all bug reports.
+If you find a security vulnerability, do NOT open an issue. Email support@sunspec.org instead.
 
-For all changes follow the procedure documented at the bottom of the [Orange Button Open Source Community Site](https://sunspec.org/ob-open-source-community/).  Our intent is to add more specific information on Continuous Integration, Vulnerability Scans, Code Style, Code Review, etc... as Orange Button Core evolves so please check back for more information.
+## How to suggest a feature or enhancement
 
-# How to report a bug
-If you find a security vulnerability, do NOT open an issue. Email support@sunspec.org instead.
+Please use the [Orange Button Slack Channel](https://orange-button.slack.com/) to suggest features and enhancements.
```

Comment:
Why not use GitHub issues for feature requests, enhancements and bug reports?  There's no tracking on slack, and it's not publically visible (you have to join the channel to browse posts) so if we use slack to collect issues, resolution will fall to those paying attention to it.

Reply:
Perhaps we switch to github issues after the first stable release.

Reply:
That sounds like a plan.  That original idea was to get people to join the workgroup as opposed to just directly adding features but a more formal procedure makes sense after the first release.